src/data/faq.json

```diff
@@ -457,7 +457,7 @@
                                 "active": false,
                                 "textblock": [
                                     "From an epidemiological point of view, a PCR test is of course significantly more accurate and informative than a self-test performed by non-professionals. Therefore, a PCR test is recommended for confirmation (or verification of any false positives) of the self-test.",
-                                    "Therefore, in order to comply with local regulations, we recommend that after a positive self-test, they contact their local health department or trusted doctor to discuss further action."
+                                    "Therefore, in order to comply with local regulations, we recommend that after a positive self-test, you contact their local health department or trusted doctor to discuss further action."

```

Comment:
should be "you contact your"

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3322/commits/ff61d079ed080cd3b21a3a6996709f3b3f04ad26, thanks!